src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react"
+import { Route, Switch } from "react-router-dom"
+import Header from "./Header"
+import InputTodo from "./InputTodo"
+import TodosList from "./TodosList";
+import { v4 as uuidv4 } from "uuid";
+
+import About from "../pages/About"
+import NotMatch from "../pages/NotMatch"
+import Navbar from "./Navbar";
+
+const TodoContainer = () => {
+
+  const [todos, setTodos] = useState(getInitialTodos())
+
+  // useEffect(() => {
+  //   // getting stored items
+  //   const temp = localStorage.getItem("todos")
+  //   const loadedTodos = JSON.parse(temp)
+
+  //   if (loadedTodos) {
+  //     setTodos(loadedTodos)
+  //   }
+  // }, [])
+
```

Comment:
- Kindly remove all inline comment in TodoContainer.js for a clean code.
- Please kindly update your readme file to add project title and description in the Readme file;